### DISCUSSION TITLE
Metodologia ágil em agências, é possível?

### DISCUSSION THREAD

**Discussion by heliomsolivas** [frontendbr/forum#862] [Discussão]
Acredito que bastante gente aqui já trabalhou em agências, e sabem como é aquela loucura diária com um projeto atropelando o outro, ficando até tarde e tudo mais. 
O que eu queria saber, é possível aplicar alguma metodologia ágil nessa "vida louca" de agência? Alguém já passou por isso? 
O que mais acontece comigo é cliente vendo o projeto quase pronto pedindo alterações de última hora e precisando lançar algo no mesmo dia. Acredito que o problema seja mais em cima, mas queria melhorar de algum jeito o começo/meio/fim dos projetos.

**Comment by kvnol** [comment]
Eu ainda trabalho em agência e se existisse a possibilidade, gostaria muito de saber.

**Comment by helderberto** [comment]
Olá @heliomsolivas!

A alguns anos, trabalhei em uma agência que utilizava _scrum_ em sua organização, eles utilizavam um ticket adicional chamado de "paraquedas" para casos de suporte, e outras tarefas urgentes, porém o _scrum master_ estava ciente sobre o novo card e que o planejamento foi mudado conforme a mudança de pauta.

O bacana com isso era que a equipe colaborava para que todos atingissem o objetivo semanal, finalizar suas tarefas, e não ficar com diversos paraquedas no _kanban_.

**Comment by klarkc** [comment]
A melhor solução é separar um time de manutenção e suporte que trabalha apenas com cards no kanban ou um sistema de tickets, do time de desenvolvimento, que procura colocar em prática o Scrum. Claro que há várias modificações no processo, para se adaptar a realidade de uma agência.

No time onde eu trabalho, cada dev fica responsável pelo feedback para o cliente e pode entrar em contato direto com o mesmo (reduzindo a dependência do PO) porém não é permitida a alteração de escopo durante o desenvolvimento, e novas solicitações são encaminhadas ao Comercial, Análise e PO.

As cerimônias ficam da mesma forma, porém o backlog grooming é feito apenas com o time encarregado (geralmente um programador e um design) e deixamos mais a cargo do próprio dev decidir quais tarefas serão puxadas do backlog para a sprint, ao final da sprint planning o PO faz uma revisão geral, retira e coloca tarefas caso necessário. Cada dev fica com a responsabilidade de se atentar ao prazo do projeto, e sempre manter a comunicação com o PO ativa, para antecipar atrasos de cronograma. Não é ainda um ideal, mas é como estamos conseguindo de certa forma nos organizar melhor, e o feedback tem sido positivo, surgindo também sempre novas modificações no processo. 

Nosso desafio atual é descobrir como orçar projetos de forma mais assertiva, e como manter um feedback para o cliente mais simplificado (estamos testando o gantt).
